sentences/sk/light_HassTurnOn.yaml

```diff
@@ -3,6 +3,13 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "(<turn_on> | <turn_on_light>) [(všetky | všetko)] [(svetlo | svetlá | osvetlenie | lampu | lampy)] [(v | vo | na | pred | pod | pri)] <area>"
+          - "(<turn_on> | <turn_on_light>) (svetlo | lampu) <area>"
```

Comment:
tato  "area" neviem ci nieje zbytocna

Reply:
Area pro jednu entitu není zbytečná, je potřebuje svůj skot pro sentences a tam definovat, ze je to jen jedno světlo.

Reply:
takze je to vporiadku? moze to takto ostat?